linalg/sparsemat.cpp

```diff
@@ -40,7 +40,7 @@ void SparseMatrix::InitCuSparse()
    // Initialize cuSPARSE library
 #ifdef MFEM_USE_CUDA
    SparseMatrixCount++;
-   if (SparseMatrixCount == 1 && Device::Allows(Backend::CUDA_MASK))
+   if (!handle && Device::Allows(Backend::CUDA_MASK))
```

Comment:
To be extra safe can we set ``cusparseHandle_t SparseMatrix::handle = nullptr``

Reply:
In the class initalizer? I like this idea.

Reply:
Yup, here 
https://github.com/mfem/mfem/blob/b9bc24bb27be115f3ad3dd3dbfa892d165ccc8a3/linalg/sparsemat.cpp#L33

Reply:
It doesn't look like the `= nullptr` update was pushed, could you double-check that @wcdawn  ?

Reply:
Should be fixed now. Thanks!